bench-show.cabal

```diff
@@ -96,6 +96,11 @@ source-repository head
     type: git
     location: https://github.com/composewell/bench-show
 
+flag wl-pretty
+  description: Use pretty printing from ansi-wl-pprint
+  manual: True
+  default: False
+
```

Comment:
Like I commented in the Graph PR, have a disable flag rather than enable. Keep this package compatible with previous versions.